### PR TITLE
[Bugfix] Memory leak fix

### PR DESCRIPTION
**Pull request by vhuseinova-msft**
## Purpose

* Updated FluentUI version
* Added missed CSS feature info in ChangeLog

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Pull request checklist

This PR has considered:
- [ ] Color Theming
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] View Data Injection
- [ ] Demo App UI/UX update

## How to Test

* Join a call
* Leave the call
* Check "Memory Graph" in Xcode

## What to Check

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| ad291be6-a55f-4c97-969e-1923b60d923d| Screen Shot 2022-11-18 at 3 28 22 PM|

### CODE REVIEW

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Azure Communication UI Calling Release History
 
+## 1.2.0-beta.1 (TBD)
+### New Features
+- Introduced CallDiagnostics to get diagnostics information for the call [#446](https://github.com/Azure/communication-ui-library-ios/pull/446)
+
```
Comment (jpeng-ms) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/CHANGELOG.md:6]: can we also document that we upgraded fluent ui to 0.8.9?

Comment (vhuseinova-msft) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/CHANGELOG.md:6]: I haven't addded it as no previous versions updates of FluentUI or CallingSDK were tracked in the ChangeLog and also, there are no changes for Contoso to see or react to add it to bug fixes.

**Review by jpeng-ms — APPROVED** [review]
Looks good!

**Review by emlynmac — APPROVED** [review]
@jimchou-dev we'll need this change merged to chat branch too.

**Pull request merged**